src/bot/api.rs

```diff
@@ -272,33 +281,41 @@ impl Bot {
         )
     }
 
-    pub fn send_video_note<C, V>(&self, chat_id: C, video_note: V) -> SendVideoNote
+    pub fn send_video_note<'a, C, V>(
+        &'a self,
+        chat_id: C,
+        video_note: V,
+    ) -> SendVideoNote
     where
-        C: Into<ChatId>,
-        V: Into<String>, // TODO: InputFile
+        C: Into<ChatId<'a>>,
+        V: Into<Cow<'a, str>>, // TODO: InputFile
     {
         SendVideoNote::new(self.ctx(), chat_id.into(), video_note.into())
```

Comment:
`SendVideoNote::new` accepts `Into`s so there is no need to call `.into()` on `Bot`'s side. (**NOTE**: it's relevant for all bot methods)